plugin/storage/cassandra/spanstore/operation_names_test.go

```diff
@@ -55,38 +69,49 @@ func withOperationNamesStorage(writeCacheTTL time.Duration, fn func(s *operation
 }
 
 func TestOperationNamesStorageWrite(t *testing.T) {
-	for _, ttl := range []time.Duration{0, time.Minute} {
-		writeCacheTTL := ttl // capture loop var
-		t.Run(fmt.Sprintf("writeCacheTTL=%v", writeCacheTTL), func(t *testing.T) {
-			withOperationNamesStorage(writeCacheTTL, func(s *operationNameStorageTest) {
+	for _, test := range []test{
+		{0, 0, nil},
+		{time.Minute, 0, nil},
+		{0, 1, nil},
+		{time.Minute, 1, nil},
```

Comment:
Please use initialization with named fields, it is very hard to understand what `0, 0, nil` is supposed to mean.

Also, add a string field `name` to the struct which describes that the sub-test is testing